### DISCUSSION TITLE
Allowed sites

### DISCUSSION THREAD

**Discussion by gojali-top** [pl4fun/jira-git-branch-name-creator#15] [Q&A] [answered]
Does this extension work on all websites which I visit?

**Comment by pl4fun** [comment, marked as answer]
Hi!
No, this script only works on the Jira(Atlassian) websites

https://github.com/pl4fun/jira-git-branch-name-creator/blob/eb24043bfd86d6da404ee49dc85d869efc9d5c0c/jira-git-branch-name-creator.js#L6
